sdk/python/tests/compiler/testdata/condition.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 kubeflow.org
+# Copyright 2021 kubeflow.org
```

Comment:
The UI has issues displaying Conditions after this update. I can fix this in a subsequent PR.